runner/src/runner/pipeline.clj

```diff
@@ -201,9 +201,15 @@
                                            [[::xt/put
                                              (assoc run-info
                                                     :status :initializing
-                                                    :started (Instant/now))]]))
+                                                    :initiated-at (Instant/now))]]))
               _ (log-event db-client run-id (str "Pulling image " image))
               _ (eng/pull-image image)
+              _ (xt/await-tx db-client
+                             (xt/submit-tx db-client
+                                           [[::xt/put
+                                             (assoc (run-info-of db-client run-id)
```

Comment:
`run-info-of` is what fixed the problem, probably I was removing the value of `initiated-at` because I was not fetching the lates value from the db and in the history the status is included because it was there at some point in time..... interesting